<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MorphologicalDictionarySimpleGui::cMainWindow</class>
 <widget class="QMainWindow" name="MorphologicalDictionarySimpleGui::cMainWindow">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCorpora Morphological Dictionary</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Search word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="wordForAnalysis_"/>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton_">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="analysisResult_">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
